src/platform/imx8/include/platform/lib/memory.h

```diff
@@ -11,6 +11,14 @@
 #define __PLATFORM_LIB_MEMORY_H__
 
 #include <sof/lib/cache.h>
+#include <config.h>
+
+/* data cache line alignment */
+#if DCACHE_LINE_SIZE > 0
```

Comment:
This should be defined in core-isa.h, so should be the same as intel DCACHE_LINE_SIZE (defined in same place).

Reply:
Indeed, and it's defined to 128. No need to make an if in this one.

Reply:
Core-isa.h are external definitions - that is why line size is used to define the cache default alignment in cache.h
I removed if in cases that only one line size is defined but we should not edit core-isa.